build.sbt

```diff
@@ -276,32 +280,61 @@ lazy val docs = http4sProject("docs")
         case _ => Seq.empty
       }
     },
+    preStageSiteDirectory := sourceDirectory.value / "hugo",
+    siteStageDirectory := target.value / "site-stage",
+    sourceDirectory in Hugo := siteStageDirectory.value,
+    watchSources := {
+      // nasty hack to remove the target directory from watched sources
+      watchSources.value
+        .filterNot(_.getAbsolutePath.startsWith(
+          target.value.getAbsolutePath))
+    },
+    copySiteToStage := {
+      streams.value.log.debug(s"copying ${preStageSiteDirectory.value} to ${siteStageDirectory.value}")
+
+      IO.copyDirectory(
+        source = preStageSiteDirectory.value,
+        target = siteStageDirectory.value,
+        overwrite = false,
+        preserveLastModified = true)
+      IO.copyDirectory(
+        source = tutTargetDirectory.value,
+        target = siteStageDirectory.value / "content" / "v0.15",
+        overwrite = false,
+        preserveLastModified = true)
+      IO.copyFile(
+        sourceFile = baseDirectory.value / ".." / "CHANGELOG.md",
+        targetFile = siteStageDirectory.value / "CHANGELOG.md",
+        preserveLastModified = true)
+    },
+    copySiteToStage <<= copySiteToStage.dependsOn(tutQuick),
+    makeSite <<= makeSite.dependsOn(copySiteToStage),
```

Comment:
These two lines cause deprecation errors in sbt-0.13.13, but I haven't quite worked out how to replace them.